doc/developers/contributing.rst

```diff
@@ -122,22 +122,26 @@ and follows the decision-making process outlined in :ref:`governance`.
 Automated Contributions Policy
 ==============================
 
+Contributing to scikit-learn requires **human judgment, contextual understanding**, and
```

Comment:
We rarely use bold (`**words here**`) in our docs, so I would not use it here either (all the changes)